src/features/metadata-instance-editor/__tests__/Instance.test.js

```diff
@@ -1,9 +1,14 @@
 import * as React from 'react';
 
-import TEMPLATE_CUSTOM_PROPERTIES from '../constants';
+import { render, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { createIntl } from 'react-intl';
```

Comment:
should not need this when you use the custom testing-library mentioned above since it should already include the intl provider